packages/app/src/systems/Network/hooks/useNetworks.ts

```diff
@@ -31,6 +32,10 @@ export function useNetworks() {
     selectors.selectedNetwork
   );
 
+  useEffect(() => {
+    store.refreshNetworks();
+  }, []);
```

Comment:
Why we need this refresh here?

Reply:
Just to ensure that the `networks` in the `store` are being refreshed and up-to-date with the data stored in the `IndexedDB`
cc: @LuizAsFight

Reply:
@LuizAsFight do we have some cache on the network list? If not I don't see why it would be needed.

Reply:
@luizstacio 
this is just to make sure that when network list opens, network list will be correct

as network machine is global, if some other machine forget to call `notifyRefreshNetworks` after change networks, the list will be incorrect.

Reply:
I don't think we should cover forgetting `notifyRefreshNetworks`. This needs to be called for an application to run correctly. And as we don't have caches every time the application is open, it should refresh.

Reply:
I don't see what's the problem, a refetch when opening a list is a pretty common behavior